certification_organism/data/product_template.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="__export__.product_template_3_cccbc90f" model="product.template" context="{'studio': True, 'create_product_product': False}">
+        <field name="project_id" ref="industry_fsm.fsm_project"/>
+        <field name="service_tracking">task_global_project</field>
+        <field name="type">service</field>
+        <field name="detailed_type">service</field>
+        <field name="uom_po_id" ref="uom.product_uom_unit"/>
+        <field name="uom_id" ref="uom.product_uom_unit"/>
+        <field name="invoice_policy">delivery</field>
+        <field name="service_type">manual</field>
+        <field name="name">Charging Station Certification</field>
+        <field name="worksheet_template_id" ref="__export__.worksheet_template_2_2529231b"/>
+        <field name="list_price">500.0</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+    </record>
+    <record id="industry_fsm_sale.field_service_product_product_template" model="product.template" context="{'studio': True, 'create_product_product': False}">
+        <field name="project_id" ref="industry_fsm.fsm_project"/>
+        <field name="service_tracking">task_global_project</field>
+        <field name="type">service</field>
+        <field name="image_1920" type="base64" file="certification_organism/static/src/binary/product_template/2-image_1920"/>
+        <field name="detailed_type">service</field>
+        <field name="uom_po_id" ref="uom.product_uom_hour"/>
+        <field name="uom_id" ref="uom.product_uom_hour"/>
+        <field name="invoice_policy">delivery</field>
+        <field name="service_type">timesheet</field>
+        <field name="name">Field Service</field>
+        <field name="list_price">100.0</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+    </record>
+    <record id="sale_timesheet.time_product_product_template" model="product.template" context="{'studio': True, 'create_product_product': False}">
```

Comment:
you can't define records with xml_id from other module, remove the prefix sale_timesheet